wdl2cwl/tests/cwl_files/BuildCembaReferences.cwl

```diff
@@ -0,0 +1,395 @@
+class: Workflow
+id: BuildCembaReferences
+inputs:
+  - id: Convert.fasta_input
+    type: File
+  - id: Convert.monitoring_script
+    type:
+      - File
+      - 'null'
+  - id: IndexForward.fasta_input
+    type: File
+  - id: IndexForward.index_prefix
+    type: string
+  - id: IndexForward.monitoring_script
+    type:
+      - File
+      - 'null'
+  - id: IndexReverse.fasta_input
+    type: File
+  - id: IndexReverse.index_prefix
+    type: string
+  - id: IndexReverse.monitoring_script
+    type:
+      - File
+      - 'null'
+  - id: CreateReferenceDictionary.reference_fasta
+    type: File
+  - id: CreateReferenceDictionary.monitoring_script
+    type:
+      - File
+      - 'null'
+  - id: CreateReferenceFastaIndex.reference_fasta
+    type: File
+  - id: CreateReferenceFastaIndex.monitoring_script
+    type:
+      - File
+      - 'null'
```

Comment:
The original WDL file has 2 defined workflow inputs, and this is reflected in the miniWDL object
```
ipdb> print(len(obj.available_inputs))
2
```

```suggestion
  - id: monitoring_script
    type:
      - File
      - 'null'
  - id: reference_fasta
    type: File
```